Examples/Examples/Demo/WebViewController.h

```diff
@@ -8,13 +8,9 @@
 
 #import <UIKit/UIKit.h>
 
-NS_ASSUME_NONNULL_BEGIN
```

Comment:
🤔 is this removal necessary?

Reply:
Whoops, forgot to add `nonnull` to the properties. Removing the `NS_ASSUME_NONNULL_BEGIN` to be more explicit.